apps/health_campaign_field_worker_app/lib/pages/inventory/stock_damaged_details.dart

```diff
@@ -134,12 +140,22 @@ class _StockReceiptDetailsPageState
                       label: localizations.translate(
                         i18.stockDetails.vehicleNumber,
                       ),
+                      maxLength: 200,
+                      isRequired: true,
+                      validationMessages: {
+                        'required': (object) => 'Vehicle number is required',
```

Comment:
In the spec, vehicle number is not required. Can you please check all the required fields once